scripts/bootstrap_env.sh

```
@@ -1,38 +1,170 @@
+    if PYTHONPATH="${PARENT_REPO}:$PYTHONPATH" python3 "$fetcher_module" fetch \
+        --github-owner "$github_owner" \
+        --github-repo "$github_repo" \
+        --output "$temp_output" \
+        ${GITHUB_PAT:+--github-token "$GITHUB_PAT"} \
+        2>/dev/null; then
+
+        # Merge with output
+        if [[ -f "$temp_output" ]]; then
+            cat "$temp_output" >> "$output_file"
+            rm -f "$temp_output"
+            local var_count=$(grep -c '^[A-Z_]=' "$temp_output" 2>/dev/null || echo "0")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)